server-spi-private/src/main/java/org/keycloak/validate/validators/EmailValidator.java

```diff
@@ -39,19 +41,18 @@ public class EmailValidator extends AbstractStringValidator implements Configure
 
     public static final String MESSAGE_INVALID_EMAIL = "error-invalid-email";
 
-
     @Override
     public String getId() {
         return ID;
     }
 
     @Override
     protected void doValidate(String value, String inputHint, ValidationContext context, ValidatorConfig config) {
-        if (!EmailValidationUtil.isValidEmail(value)) {
+        if (!EmailAddressValidator.isValidStrict(value)) {
```

Comment:
Has anybody tried running this checker against the list of current user email addresses on live? I wonder how many addresses might break because of this in the future.